Assets/Scripts/DataCollection/ConditionSwitcher.cs

```diff
@@ -65,13 +65,19 @@ private IEnumerator ChangeConditionAndLoadSecondCondition()
         Debug.Log("Changed condition. Is DDA condition: " + conditionSetter.IsDDACondition());
 
         UnlockPlayersMouse();
+        StopMusic();
 
         if (PhotonNetwork.IsMasterClient)
         {
             PhotonNetwork.LoadLevel(secondConditionSceneName);
         }
         yield return null;
     }
+    private void StopMusic()
+    {
+        List<GameObject> players = new List<GameObject>(GameObject.FindGameObjectsWithTag("Player"));
+        players.ForEach(player => player.GetComponent<PlayerManager>().DisableMusic());
+    }
```

Comment:
Can we reuse the players that we have to find for both `UnlockPlayersMouse` and `StopMusic`? Pass the list of them to the functions, so no need to find twice

Reply:
Good idea